src/stream_base.cc

```diff
@@ -100,94 +100,118 @@ int StreamBase::Writev(const FunctionCallbackInfo<Value>& args) {
 
   Local<Object> req_wrap_obj = args[0].As<Object>();
   Local<Array> chunks = args[1].As<Array>();
+  bool all_buffers = args[2]->BooleanValue();
```

Comment:
Can you use the overload that returns a `Maybe<bool>`, or alternatively use `IsTrue()` instead?